src/server/routes.rs

```diff
@@ -26,6 +26,11 @@ pub fn config<T: Store + 'static>(cfg: &mut ServiceConfig) {
                 resource("/login")
                     .route(get().to(handlers::auth::login_info))
                     .route(post().to(handlers::auth::login::<T>)),
+            )
+            .service(
+                resource("/account/whoami")
+                    .route(get().to(handlers::account::whoami))
+                    .wrap(AuthChecker::new()),
```

Comment:
Probably do rate limiting on a per resource basis, it seems it should be possible, but need to look more into it.

Reply:
it is definitely possible, and necessary. many endpoints shouldn't count against rate limits. biggest concern is brute force attempts against login endpoint

Reply:
We can revisit limiting at a later time.